libs/libc/misc/Make.defs

```diff
@@ -22,6 +22,7 @@
 
 CSRCS += lib_umask.c lib_utsname.c
 CSRCS += lib_xorshift128.c lib_tea_encrypt.c lib_tea_decrypt.c
+CSRCS += lib_err.c
```

Comment:
Can this be behind a Kconfig to prevent collisions with preexisting code?

Reply:
i think compiler optimization do it better?

Reply:
My concern is not about unused function. It is about duplication of symbol names 

```
err
verr
errx
verrx
```
used in NuttX based system before this PR that have different semantics.

If a user of NuttX uses **their** this will cause link failure.

Reply:
@davids5 how about the new implementation?